tfm-plan.sh

```diff
@@ -77,7 +79,11 @@ cd $folder
     if [ "$destroy_mode" = "true" ]; then 
         terraform plan -destroy -var-file="$tfvars_file" -out="$tfplan_output" -detailed-exitcode
     else
-        terraform plan -var-file="$tfvars_file" -out="$tfplan_output" -detailed-exitcode
+        replaced=()
+        for replaced_resource in ${replaced_resources[@]}; do
+          replaced+=("--replace=$replaced_resource")
+        done
+        terraform plan -var-file="$tfvars_file" -out="$tfplan_output" -detailed-exitcode "${replaced[@]}"
```

Comment:
in case we want to continue with this, bash linter complains about `${replaced_resources[@]}`, it wants it to be in double quotes, but I actually want to treat `$replaced_resources` string as an array here (split by spaces). Not sure how this should be properly done in bash

Reply:
based on 
https://www.cyberciti.biz/faq/bash-for-loop-array/
using quotes
![image](https://user-images.githubusercontent.com/56644044/225064679-17b88760-4b1b-4bb6-b63f-373279943c49.png)

Reply:
yes, it would work if we had an array of strings like this `('resource1', 'resource2')`
but we have a string `'resource1 resource2'`, so in your example it would be interpreted as an array containing one element (same string `'resource1 resource2'`)